zenodo_uploader.py

```diff
@@ -0,0 +1,16 @@
+import requests
+import os
+os.getenv('ZENODO_ACCESS_TOKEN: ')
+headers = {"Content-Type": "application/json"}
+params = {'access_token': ACCESS_TOKEN}
+r = requests.post('https://sandbox.zenodo.org/api/deposit/depositions', params=params, json={}, headers=headers)
+bucket_url = r.json()["links"]["bucket"]
+filename = input("Enter filename as 'my-file.zip': ")
+path = input("Enter file path as '/path/to/my-file.zip': ")
+print("Hold on. This will take a while...")
+with open(path, "rb") as fp:
+    r = requests.put(
+        "f"(bucket_url, filename),
```

Comment:
That's the correct syntax. Please read the docs about Python f-strings I posted earlier.
```suggestion
        f"{bucket_url}/{filename}",
```

Reply:
Sorry! I should have updated this sooner. I made significant changes to this file already.